src/main/resources/templates/fragments/layout.html

```diff
@@ -62,6 +62,13 @@
                           <span>List of Owners</span>
                       </li>
                 </th:block>
+				
+				<th:block th:if="${toggles[0] == true || toggles[2] == true || toggles[3] == true}">
```

Comment:
Remove this li and instead add the ` || toggles[3] == true` to the above find owners menu item (line 52)